examples/video-chat/screens/PreviewScreen/PreviewScreen.tsx

```diff
@@ -101,9 +102,13 @@ function PreviewScreen({
   }, [getCaptureDevices, startCamera]);
 
   const onJoinPressed = async () => {
-    await connect(route.params.fishjamUrl, route.params.peerToken, {
-      name: route.params.userName,
-    });
+    await connect<ParticipiantMetadata>(
+      route.params.fishjamUrl,
+      route.params.peerToken,
```

Comment:
I think `route.params.peerToken` be renamed to `route.params.participant`

Reply:
That is interesting problem: how we should call peer token now?